app/shared/uptime_service.py

```diff
@@ -0,0 +1,160 @@
+from app.models.lift import Lift
+
+from datetime import datetime
+import os
+
+
+SEASON = os.getenv('SEASON', 'Winter')
+
+# Lift schedules follow a Monday - Sunday order
+# with open and close times in 24H style
+# in UTC
+
+SUMMER_LIFT_SCHEDULES = {
+    'Panorama Lower': [
+        [9, 17],
+        [9, 17],
+        [9, 17],
+        [9, 17],
+        [9, 17],
+        [9, 17],
+        [9, 17]
+    ],
+    'Panorama Upper': [
+        [9, 17],
+        [9, 17],
+        [9, 17],
+        [9, 17],
+        [9, 17],
+        [9, 17],
+        [9, 17]
+    ],
+    'Discovery Express 11': [
+        [9, 17],
+        [9, 17],
+        [9, 17],
+        [9, 17],
+        [9, 17],
+        [9, 17],
+        [9, 17]
+    ],
+    'Canyon Express 16': [
+        [0, 0],
+        [0, 0],
+        [0, 0],
+        [0, 0],
+        [10, 18],
+        [10, 18],
+        [10, 18]
+    ],
+    'Stump Alley Express 2': [
+        [0, 0],
+        [0, 0],
+        [0, 0],
+        [0, 0],
+        [10, 18],
+        [10, 18],
+        [10, 18]
+    ]
+}
+
+# TODO: Will have to figure this out when they post it
+WINTER_LIFT_SCHEDULES = {}
```

Comment:
Mammoth says every lift and trail should be 8:30AM-4PM PST.

These should have defaults so that if a lift can't be found, a default is used.